test/integration.test.js

```diff
@@ -1,6 +1,5 @@
 const { Account, KeyPair, Near, InMemoryKeyStore } = require('../');
-const  { aliceAccountName, newAccountCodeHash, storageAccountIdKey, createFakeStorage, sleep } = require('./test-utils');
-const dev = require('../dev');
+const test_utils = require('./test-utils');
```

Comment:
s/test_utils/testUtils/g

Reply:
Should the file also be named camel case style?

Reply:
@ilblackdragon I think dash is more common for module names.